### ISSUE TITLE
ON Android 13 when redirect fires I get a pick app (Chrome and my app)

### ISSUE THREAD

**Issue by billnbell** [FormidableLabs/react-native-app-auth#848]
This is new when I upgrade to latest Android 13.0

![image](https://user-images.githubusercontent.com/25377089/236310725-652718de-91f9-4da4-8677-54c6d9bc2696.png)

I get a popup "Disambiguation" ? 

![image](https://user-images.githubusercontent.com/25377089/236310767-4cf84e1b-fc43-4d96-9003-a1f8d5b1a485.png)

What can we do to remove that?

**Comment by billnbell** [comment]
By the way this does not happen on Pixel 5/Android 12.0/and Api 31.

This is new to our customers.

**Comment by kabirbaidhya** [comment]
Not sure if this helps but I observed similar issue with native "Open with" prompt showing up on redirection in android which was due to improper activity/intent configuration. 
After I followed these [instructions](https://github.com/FormidableLabs/react-native-app-auth#android-setup) and more specifically what was recommended in [this comment](https://github.com/FormidableLabs/react-native-app-auth/issues/494#issuecomment-797394994), my issue was resolved.

You can check that if you haven't already.

**Comment by billnbell** [comment]
Yes I did that. And I tried each way. This is a different issue in Android 13. It appears my app is not registering my scheme as 'theclubsfdc'... I am getting one from Chrome and one from My app as disambiguation "intent". See above. When I set up the auth wrong I get 2 of the same app. Unsure how to tell chrome to ignore the scheme "theclubsfdc" ?

**Comment by billnbell** [comment]
I tried adding autoVerify like below on intent-filters, but it did not help me.

```
<intent-filter android:autoVerify="true">
```
